### ISSUE TITLE
Prerequisites unclear / error getting location data

### ISSUE THREAD

**Issue by Gnarfoz** [db-EV/ZoePHP#32]
The README mentions an _active data subscription_ as a prerequisite.
There's two offers from Renault, though. One for basic data services, and the other one is for navigation and live traffic data.
Are both required? I only have the first one.

I seem to be getting an error when the script tries to fetch the location:
```
{
    "type": "TECHNICAL",
    "messages": [{
            "code": "err.tech.502",
            "message": "{\"errors\":[{\"status\":\"502\",\"code\":\"gateway.processing\",\"title\":\"VNEXT\",\"detail\":\"an internal error occured while processing request : 403 FORBIDDEN\"}]}"
        }
    ],
    "errors": [{
            "errorCode": "err.tech.502",
            "errorMessage": "{\"errors\":[{\"status\":\"502\",\"code\":\"gateway.processing\",\"title\":\"VNEXT\",\"detail\":\"an internal error occured while processing request : 403 FORBIDDEN\"}]}"
        }
    ],
    "error_reference": "TECHNICAL"
}
```

This also blocks writing to the database file, because the location data is tied into the overall success check.

**Comment by Gnarfoz** [comment]
I see. This also impacts the temperature/weather information, since the GPS data is used as an input for the API call for that.

**Comment by db-EV** [comment]
I've added a fix for this. Can you try the updated [index.php](url=https://github.com/db-EV/ZoePHP/blob/main/src/index.php) and give me feedback if it behavs like expected?

**Comment by db-EV** [comment]
Changed database writing behavior a little bit more. Waiting for your feedback before publishing a new version.

**Comment by Gnarfoz** [comment]
That seems to do the trick! Thanks for the quick update.

**Comment by db-EV** [comment]
Fixed in release [20240101](https://github.com/db-EV/ZoePHP/releases/tag/20230418). Thanks for reporting and testing!

**Comment by Gnarfoz** [comment]
Thanks for the cool script! And a happy new year as well. 🎉
